### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Runquery_ScreenReader: ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#180]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**#GitHub Tags:**
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG1.3.1; #Linux; #Element:TabItem; #ScreenReader; #ORCA; #External; #External:AzureDataStudio; #FirstParty:22673;

**Actual Result:​**
ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab.

**Expected Result:​**
ORCA screen reader should announce "17:14:34" which appears under the 'Messages' tab.

**Environment Details:​**
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

**Repro Steps:**

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Run" button and hit ENTER key. Start screen reader.
5. TAB to "Messages" tab item and observe that ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab.

### User Impact:
Screen reader users will not get information about the "17:14:34" which appears under the 'Messages' tab.

### Attachment:
[Video 3-30 at 12.37 (5).webm](https://user-images.githubusercontent.com/90901519/228828239-beb2b941-aef1-43d9-a7b9-1eec6400c393.webm)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by PoojaNamde** [comment]
@nitishsharma25

As per the external bug process we have to log external bugs to MySQL extension. Please reactivate all the bugs and share all the details like VSO, area path details to log an external bug

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/22673

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-06-23

Verified this issue in the latest env Version: 1.45.0-insider
Commit: https://github.com/microsoft/azuredatastudio/commit/d414c8de56a189b5f85c28ba18d9704ce62c7ba0
Date: 2023-06-05T05:46:00.131Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.19.0-43-generic.

Issue is fixed, hence closing the bug.

[Video 6-06 at 12.24.webm](https://github.com/microsoft/azuredatastudio-mysql/assets/90901519/b2ee82c8-195f-4acd-8f2c-0ed657978a27)
